libs/@guardian/source-react-components/src/button/styles.ts

```diff
@@ -303,8 +303,8 @@ export const buttonStyles =
 		button,
 		sizes[size],
 		priorities[priority](theme.button),
-		iconSvg || isLoading ? iconSizes[size] : '',
-		(iconSvg || isLoading) && !hideLabel ? iconSides[iconSide] : '',
```

Comment:
if iconSvg is `false` or `0` or `''` and isLoading is `true` this will still evaluate to false now